test/unit/utils/test_utils.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Fairlearn contributors.
+# Licensed under the MIT License.
+
+import numpy as np
+import pandas as pd
+
+import fairlearn.utils._input_validation as iv
+
+
+class TestValidateAndReformatInput:
+    def test_smoke(self):
```

Comment:
it'd be nice to have a comment like `regression test for #906`

Reply:
Done